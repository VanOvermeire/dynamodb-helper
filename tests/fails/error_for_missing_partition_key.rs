use dynamodb_helper::DynamoDb;

#[derive(DynamoDb)]
pub struct Example {
    first: String,
    second: u32,
}

fn main() {}
