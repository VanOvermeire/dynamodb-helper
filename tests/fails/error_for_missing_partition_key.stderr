error: You need to define a partition key for your DynamoDB struct! Place the `#[partition]` attribute above the field that serves as your key.
 --> tests/fails/error_for_missing_partition_key.rs:4:12
  |
4 | pub struct Example {
  |            ^^^^^^^
